src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunner.cs

```diff
@@ -174,14 +173,14 @@ private void DetectTestFramework(ICollection<TestCase> tests)
         }
 
         private TestRunResult RunVsTest(IEnumerable<TestCase> testCases, int? timeoutMs,
-            Dictionary<string, string> envVars)
+            IDictionary<string, string> envVars)
         {
-            var testResults = RunAllTests(testCases, envVars, GenerateRunSettings(timeoutMs, false), false);
+            var (testResults, aborted) = RunAllTests(testCases, envVars, GenerateRunSettings(timeoutMs, false), false);
 
             // For now we need to throw an OperationCanceledException when a testrun has timed out. 
             // We know the test run has timed out because we received less test results from the test run than there are test cases in the unit test project.
             var resultAsArray = testResults as TestResult[] ?? testResults.ToArray();
-            if (resultAsArray.All(x => x.Outcome != TestOutcome.Failed) && resultAsArray.Count() < (testCases ?? _discoveredTests).Count())
+            if (resultAsArray.All(x => x.Outcome != TestOutcome.Failed) && aborted/*resultAsArray.Count() < (testCases ?? _discoveredTests).Count()*/)
```

Comment:
Could you delete the commented out code?

Reply:
yep, should have done it. fixed in latest commit